sorbet/rbi/gems/activemodel@5.2.4.3.rbi

```diff
@@ -1,6 +1,6 @@
 # DO NOT EDIT MANUALLY
 # This is an autogenerated file for types exported from the `activemodel` gem.
-# Please instead update this file by running `tapioca sync`.
+# Please instead update this file by running `tapioca generate activemodel`.
```

Comment:
Let's keep this `tapioca sync` please

Reply:
I guess because I used `bundle exec tapioca generate` command, this text has been modified, I didn't change it manually. I guess I need to use `sync` command  instead of `generate` command ?

Reply:
you can manually update this line, but generally it is better to delete the RBI file and run `tapioca sync` instead.